mappings/net/minecraft/client/gui/widget/MultilineScrollableWidget.mapping

```diff
@@ -6,5 +6,6 @@ CLASS net/minecraft/unmapped/C_qnehaoor net/minecraft/client/gui/widget/Multilin
 		ARG 3 width
 		ARG 4 height
 		ARG 5 text
+	METHOD m_harbhiqe overflowing ()Z
```

Comment:
```suggestion
	METHOD m_harbhiqe isScrollbarVisible ()Z
```
by convention this would be `isOverflowing`, but since that name's taken, I looked into mojang's name, and it's `showingScrollBar`. I think this is a good place to start since it's usage is in an `if` of whether or not to subtract `getScrolbarHeight` from the width of a widget. 

I also noticed that `getScrollbarHeight` should be `getScrolbarWidth`, could that fix be added to this PR?